documentation/guide-access-control.asciidoc

```diff
@@ -14,73 +14,18 @@ Definition:
 
 The one authenticated is properly called _subject_ or http://docs.oracle.com/javase/7/docs/api/java/security/Principal.html[_principal_]. However, for simplicity we use the common term _user_ even though it may not be a human (e.g. in case of a service call from an external system).
 
-To prove his authenticity the user provides some secret called _credentials_. The most simple form of credentials is a password.
+There are two forms of principals you need to distinguish while designing your authentication: `human users` and `autonomous systems`. While e.g. a Kerberos/SPNEGO Single-Sign-On makes sense for human users, it is pointless for authenticating autonomous systems.
```

Comment:
This is a bit contrary to the sentence before, where we say that "we use the common term **user**"

Reply:
Agreed. It sounds kind of. But the 2 principal forms are important to distinguish while DESIGN the authentication. When mentioning, it can simplify all as "user".

Reply:
Maybe you can combine it with the sentence before and then say in the last sentence that we use the common term user for all types